components/scream/src/physics/rrtmgp/CMakeLists.txt

```diff
@@ -1,50 +1,59 @@
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rrtmgp/mo_gas_concentrations.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rrtmgp/mo_gas_optics.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rrtmgp/mo_gas_optics_rrtmgp.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rrtmgp/mo_rrtmgp_constants.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rrtmgp/mo_rrtmgp_util_reorder.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rrtmgp/mo_rrtmgp_util_string.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rte/mo_fluxes.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rte/mo_optical_props.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rte/mo_rte_kind.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rte/mo_rte_lw.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rte/mo_rte_sw.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rte/mo_rte_util_array.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rte/mo_source_functions.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rte/kernels/mo_fluxes_broadband_kernels.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rte/kernels/mo_optical_props_kernels.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rte/kernels/mo_rte_solver_kernels.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/extensions/mo_fluxes_byband.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/extensions/mo_fluxes_byband_kernels.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/extensions/mo_fluxes_bygpoint.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/extensions/mo_heating_rates.F90
-  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/extensions/mo_rrtmgp_clr_all_sky.F90
+  rrtmgp_inputs_initializer.cpp
+  tests/rrtmgp_test_utils.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rrtmgp/kernels/mo_gas_optics_kernels.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rrtmgp/mo_rrtmgp_constants.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rrtmgp/mo_rrtmgp_util_reorder.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rte/mo_rte_lw.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rte/mo_rte_sw.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rte/expand_and_transpose.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rte/kernels/mo_fluxes_broadband_kernels.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rte/kernels/mo_optical_props_kernels.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/rte/kernels/mo_rte_solver_kernels.cpp
+  ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/examples/mo_load_coefficients.cpp
+  # This is a hack; I should not have to build this here, but we need it for the standalone RRTMGP
```

Comment:
Is this comment referring to the line above? If so, it's customary (in my experience, at least) to place the comment above what it's referring to.

Reply:
Ahh, looks like it might pertain to the stuff below. Maybe this just has to do with GitHub's code review interface.

Reply:
It refers to the hack below :-)